test/test_models.py

```diff
@@ -0,0 +1,131 @@
+from freezegun import freeze_time
+
+from src.models import (
+    User,
+    SuperUser,
+    Activity,
+    ActivityTypes,
+    SuperUserRoles,
+    create_user,
+    create_superuser,
+    create_activity,
+)
+import pytest
+from datetime import datetime
+from pydantic import ValidationError
+
+mock_incomplete_data_user = {
+    "user_id": 1,
+    "username": "Pippo",
+    "email": "aaa@bbb.cc",
+}
+
+mock_data_user = {
+    "username": "Pippo",
+    "email": "aaa@bbb.cc",
+    "age": 23,
+    "country": "GB",
+}
+
+mock_data_superuser = {
+    "username": "Pippo",
+    "email": "aaa@bbb.cc",
+    "age": 23,
+    "country": "GB",
+    "role": SuperUserRoles.admin,
+}
```

Comment:
I'd recommend to use [pytest fixtures](https://docs.pytest.org/en/stable/explanation/fixtures.html) instead of global variables if you want to use same mock data for the tests :)

Reply:
Added! Check if it looks ok now